src/main/java/org/encog/util/normalize/output/OutputFieldRangeMapped.java

```diff
@@ -101,10 +101,13 @@ public OutputFieldRangeMapped(InputField f) {
 	 * @return The calculated value.
 	 */
 	public double calculate(final int subfield) {
-		return ((this.field.getCurrentValue() - this.field.getMin()) 
-				/ (this.field
-				.getMax() - this.field.getMin()))
-				* (this.high - this.low) + this.low;
+		if (this.field.getMax() == this.field.getMin()) {
+			return 0;
+		} else {
+			return ((this.field.getCurrentValue() - this.field.getMin()) 
+					/ (this.field.getMax() - this.field.getMin()))
```

Comment:
`this.field` should have method that return normalized value to make code more redable